lib/logstash/filters/ruby.rb

```diff
@@ -43,7 +43,11 @@ def filter(event,&block)
       @codeblock.call(event,&block)
       filter_matched(event)
     rescue Exception => e
-      @logger.error("Ruby exception occurred: #{e}")
+      err_details = {}
+      err_details["exception"] = e
+      err_details["event"] = event
+      err_details["code"] = @code
+      @logger.error("Ruby exception occurred: #{err_details}")
```

Comment:
In logstash we tend to use a common format to log structured data like in this scenario. Can you please change to the following?

```ruby
@logger.error("Ruby exception occurred.", :exception => e.class.name, :message => e.message, :backtrace => e.backtrace)
```